<launch>
  <include file="/opt/ros/foxy/share/rosbridge_server/launch/rosbridge_websocket_launch.xml"/>
  <node pkg="cysar" exec="teleop.py" name="teleop">
    <param name="deadzone" value="0.08"/>
    <param name="max_speed" value="1.0"/>
    <param name="flipper_sensitivity" value="1.0"/>
    <param name="flipper_min" value="-14.0"/>
    <param name="flipper_max" value="14.0"/>
    <param name="flipper_max" value="14.0"/>
    <param name="shoulder_rotation_min" value="-1.0"/>
    <param name="shoulder_rotation_max" value="1.0"/>
    <param name="shoulder_rotation_sensitivity" value="0.1"/>
    <param name="shoulder_angle_min" value="-1.0"/>
    <param name="shoulder_angle_max" value="1.0"/>
    <param name="shoulder_angle_sensitivity" value="0.1"/>
    <param name="elbow_angle_min" value="-1.0"/>
    <param name="elbow_angle_max" value="1.0"/>
    <param name="elbow_angle_sensitivity" value="0.1"/>
    <param name="wrist_rotation_min" value="-1.0"/>
    <param name="wrist_rotation_max" value="1.0"/>
    <param name="wrist_rotation_sensitivity" value="0.1"/>
    <param name="wrist_angle_min" value="-1.0"/>
    <param name="wrist_angle_max" value="1.0"/>
    <param name="wrist_angle_sensitivity" value="0.1"/>
    <param name="claw_closure_min" value="-1.0"/>
    <param name="claw_closure_max" value="1.0"/>
    <param name="claw_closure_sensitivity" value="0.1"/>
  </node>
  <node pkg="cysar" exec="operator_interface.py" name="operator_interface"/>
  <node pkg="cysar" exec="camera_control.py" name="camera_control"/>
  <!--<node pkg="cysar" exec="can_control.py" name="can_control"/>-->
</launch>
